tests/metrics/column_aggregate/test_column_aggregate.py

```diff
@@ -0,0 +1,106 @@
+from pathlib import Path
+
+import pandas
+import pytest
+from pytest import FixtureRequest
+
+from great_expectations.metrics.column_aggregate.mean import (
+    ColumnValuesMean,
+    ColumnValuesMeanResult,
+)
+from great_expectations.metrics.metric_results import MetricErrorResult
+from tests.integration.test_utils.data_source_config import (
+    PandasDataFrameDatasourceTestConfig,
+    PostgreSQLDatasourceTestConfig,
+    SnowflakeDatasourceTestConfig,
+    SparkFilesystemCsvDatasourceTestConfig,
+)
+from tests.integration.test_utils.data_source_config.pandas_data_frame import (
+    PandasDataFrameBatchTestSetup,
+)
+from tests.integration.test_utils.data_source_config.postgres import PostgresBatchTestSetup
+from tests.integration.test_utils.data_source_config.snowflake import (
+    SnowflakeBatchTestSetup,
+)
+from tests.integration.test_utils.data_source_config.spark_filesystem_csv import (
+    SparkFilesystemCsvBatchTestSetup,
+)
+
+
+@pytest.fixture
+def dataframe() -> pandas.DataFrame:
+    return pandas.DataFrame(
+        {
+            "id": [1, 2, 3, 4],
+            "number": [1, 2, 3, 4],
+            "string": ["a", "b", "c", "d"],
+        },
+    )
+
+
+@pytest.fixture
+def setup_pandas(dataframe: pandas.DataFrame) -> PandasDataFrameBatchTestSetup:
+    return PandasDataFrameBatchTestSetup(
+        config=PandasDataFrameDatasourceTestConfig(),
+        data=dataframe,
+    )
+
+
+@pytest.fixture
+def setup_spark(dataframe: pandas.DataFrame, tmp_path: Path) -> SparkFilesystemCsvBatchTestSetup:
+    return SparkFilesystemCsvBatchTestSetup(
+        config=SparkFilesystemCsvDatasourceTestConfig(),
+        data=dataframe,
+        base_dir=tmp_path,
+    )
+
+
+@pytest.fixture
+def setup_postgres(dataframe: pandas.DataFrame) -> PostgresBatchTestSetup:
+    return PostgresBatchTestSetup(
+        config=PostgreSQLDatasourceTestConfig(), data=dataframe, extra_data={}
+    )
+
+
+@pytest.fixture
+def setup_snowflake(dataframe: pandas.DataFrame) -> SnowflakeBatchTestSetup:
+    return SnowflakeBatchTestSetup(
+        config=SnowflakeDatasourceTestConfig(), data=dataframe, extra_data={}
+    )
+
+
+@pytest.mark.parametrize(
```

Comment:
I just realized I can probably reduce the fixture code in this file if I use our `parameterize_batch_for_data_sources` fixture.

Reply:
yes - this looks good, but i would suggest we rewrite the tests to use that fixture. should be pretty easy and a negative diff.

Reply:
Done.